recipes/sail/all/conanfile.py

```diff
@@ -32,25 +33,15 @@ class SAILConan(ConanFile):
         "with_avif": True,
         "with_gif": True,
         "with_jpeg2000": True,
-        "with_jpeg": True,
+        "with_jpeg": "libjpeg",
         "with_png": True,
         "with_tiff": True,
         "with_webp": True,
     }
-    generators = "cmake", "cmake_find_package", "cmake_find_package_multi"
-
-    @property
-    def _source_subfolder(self):
-        return "src"
-
-    @property
-    def _build_subfolder(self):
-        return "build"
 
     def export_sources(self):
-        self.copy("CMakeLists.txt")
-        for patch in self.conan_data.get("patches", {}).get(self.version, []):
-            self.copy(patch["patch_file"])
+        for patch_file in self.conan_data.get("patches", {}).get(self.version, []):
+            copy(self, patch_file["patch_file"], self.recipe_folder, self.export_sources_folder)
```

Comment:
```suggestion
        export_conandata_patches(self)
```
Same idiom, simplified by Conan tools